### PR TITLE
Revising separator feature [stage-2]

### PR DESCRIPTION
**Pull request by stulees**
- Show/hide row and column separators independently
- Remove setting for separator size
- e2e and integration tests revised

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review, thanks

File: src/widget/css/fixed-data-table-overrides.css [hunk 1]
```diff
@@ -16,27 +16,46 @@
...
+
+.public_fixedDataTableCell_main {
+  border-color: transparent;
+}
+
+.public_fixedDataTableColumnResizerLine_main {
+  border-color: transparent;
+}
+
+.public_fixedDataTableRow_fixedColumnsDivider {
```
Comment (donnapep) [inline src/widget/css/fixed-data-table-overrides.css]: @stulees Since all these selectors are applying the same style, can they be grouped together?

Comment (stulees) [inline src/widget/css/fixed-data-table-overrides.css]: @donnapep They could but I kept them separate to follow Fixed Data Tables applying of their class styles and also for clarity. Do you prefer them grouped?

Comment (donnapep) [inline src/widget/css/fixed-data-table-overrides.css]: @stulees I guess it doesn't matter. It just seems like a bad practice.

**Comment by donnapep** [comment]
👍

**Pull request merged**